it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersLtdLiabilityPartnershipControllerISpec.scala

```diff
@@ -9,20 +10,22 @@ class BusinessPartnersLtdLiabilityPartnershipControllerISpec
   extends Specifications with TestConfiguration {
 
   "GET /form/business-partners/limited-liability-partnership-name" should {
-    "render the Limited Liability Partnership Name page" in {
-      given
-        .commonPrecondition
+    "the limited liability partnership name is entered" should {
+      "return 200 with ltdLiabilityPartnershipName" in {
+        given
+          .commonPrecondition
 
-      WsTestClient.withClient { client =>
-        val result = client.url(s"$baseUrl/form/business-partners/limited-liability-partnership-name")
-          .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
-          .get()
+        WsTestClient.withClient { client =>
+          val result = client.url(s"$baseUrl/form/business-partners/limited-liability-partnership-name")
+            .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
+            .withHttpHeaders(xSessionId,
+              "Csrf-Token" -> "nocheck")
+            .post(Map("ltdLiabilityPartnershipName" -> Seq("Partnership Name")))
```

Comment:
Should this test be in the POST section ?

Reply:
Yes, this meant to be the post request, must have made the mistake when i rebasing. I've reverted it back to a GET.